.gitignore

```diff
@@ -0,0 +1,51 @@
+# Compiled source #
+###################
+*.com
+*.class
+*.dll
+*.exe
+*.o
+*.so
+*.pyc
+
+# Packages #
+############
+# it's better to unpack these files and commit the raw source
+# git has its own built in compression methods
+*.7z
+*.dmg
+*.gz
+*.iso
+*.jar
+*.rar
+*.tar
+*.zip
+
+# Logs and databases #
+######################
+*.log
+*.sql
+*.sqlite
+
+# LaTeX Generated Files #
```

Comment:
I'll fix it up and resubmit. I'm still learning git and this is a wild ride.

On Tue, May 17, 2016 at 5:17 PM Will Benton <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/willb/wi-thesis-template/pull/3#discussion_r63614272>:
>
> > +*.7z
> > +*.dmg
> > +*.gz
> > +*.iso
> > +*.jar
> > +*.rar
> > +*.tar
> > +*.zip
> > +
> > +# Logs and databases #
> > +######################
> > +*.log
> > +*.sql
> > +*.sqlite
> > +
> > +# LaTeX Generated Files #
>
> Yes: it's not an issue of boilerplate, it's an issue of surprising users
> who might actually want to commit those things. In general, it's better to
> tread lightly with default .gitignores. :-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/willb/wi-thesis-template/pull/3/files/b9852d04476cd951b43e969e4ba8bbc75b1b6fb1..642934b69daba76b188d9e1687c3895e09617080#r63614272>
>